### ISSUE TITLE
generic error when using config on py-editor setup

### ISSUE THREAD

**Issue by MarcSkovMadsen** [pyscript/pyscript#2119]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?

![image](https://github.com/user-attachments/assets/53ab1303-0b32-4167-ba17-44e19685d826)

### What browsers are you seeing the problem on? (if applicable)

Microsoft Edge

### Console info

```shell
MessageEvent {type: 'error', error: <generic error, no stack>}
```

### Additional Context

**mini-coi.js**

```javascript
/*! coi-serviceworker v0.1.7 - Guido Zuidhof and contributors, licensed under MIT */
/*! mini-coi - Andrea Giammarchi and contributors, licensed under MIT */
(({ document: d, navigator: { serviceWorker: s } }) => {
    if (d) {
      const { currentScript: c } = d;
      s.register(c.src, { scope: c.getAttribute('scope') || '.' }).then(r => {
        r.addEventListener('updatefound', () => location.reload());
        if (r.active && !s.controller) location.reload();
      });
    }
    else {
      addEventListener('install', () => skipWaiting());
      addEventListener('activate', e => e.waitUntil(clients.claim()));
      addEventListener('fetch', e => {
        const { request: r } = e;
        if (r.cache === 'only-if-cached' && r.mode !== 'same-origin') return;
        e.respondWith(fetch(r).then(r => {
          const { body, status, statusText } = r;
          if (!status || status > 399) return r;
          const h = new Headers(r.headers);
          h.set('Cross-Origin-Opener-Policy', 'same-origin');
          h.set('Cross-Origin-Embedder-Policy', 'require-corp');
          h.set('Cross-Origin-Resource-Policy', 'cross-origin');
          return new Response(body, { status, statusText, headers: h });
        }));
      });
    }
  })(self);
```

**config.toml**

```toml
packages = [
    "https://cdn.holoviz.org/panel/1.4.4/dist/wheels/bokeh-3.4.1-py3-none-any.whl",
    "https://cdn.holoviz.org/panel/1.4.4/dist/wheels/panel-1.4.4-py3-none-any.whl"
]
```

**script.html**

```html
<!DOCTYPE html>
<html>
  <head>
    <script src="./mini-coi.js" scope="./"></script>   
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width,initial-scale=1" />
    <title>🦜 Panel Editor Template</title>
    <link rel="stylesheet" href="https://pyscript.net/releases/2024.7.1/core.css">  
    <script type="module" src="https://pyscript.net/releases/2024.7.1/core.js"></script>
  </head>
  <body>
    <script type="mpy-editor" config="./config.toml" env="shared" setup>
        import panel as pn

        pn.extension(sizing_mode="stretch_width")
    </script>
    <script type="py-editor" env="shared">
        slider = pn.widgets.FloatSlider(start=0, end=10, name='Amplitude')
  
        def callback(new):
           return f'Amplitude is: {new}'
  
        pn.Row(slider, pn.bind(callback, slider)).servable(target="out");
    </script>
    <div id="out"></div>
  </body>
</html>
```

**Comment by WebReflection** [comment]
is this a duplicate of this? https://github.com/pyscript/pyscript/issues/2118

**Comment by MarcSkovMadsen** [comment]
No. Its a seperate issue.

**Comment by WebReflection** [comment]
sorry for late come back .... but ...

  * you cannot share the env between micropython and pyodide ... these are two completely different worlds, PLs, WASM engines, and so on if you use a setup for *pyodide* be sure its `type="py-editor"` and **not mpyeditor**.
  * there were issues with bokeh and panel fixed upstream but we also released 2024.8.1 so please double check you are dong the right thing (same interpreter per shared env / setup) and you are using latest bokeh and panel, thanks

**Comment by WebReflection** [comment]
btw, about setup errors, this would fix but still you can't have `mpy-editor` setup for `py-editor` content, I hope that's clear (or it was a typo in the original post) https://github.com/pyscript/pyscript/pull/2138
